### PR TITLE
Issue #SC-1474 fix: Stronger rules for passwords

### PR DESCRIPTION
**Pull request by ArjunKhetia-NIIT**

### CODE REVIEW

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts [hunk 1]
```diff
@@ -150,17 +150,13 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
...
-      } else if (!lwcsRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.lwcsError;
-      } else if (!upcsRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.upcsError;
-      } else if (!numRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.numError;
-      } else if (specRegex.test(val)) {
-        this.passwordError = this.resourceService.frmelmnts.lbl.passwd.specError;
+      const specRegex = new RegExp('^[^<>{}\'\"/|;:.\ ,~!?@#$%^=&*\\]\\\\()\\[¿§«»ω⊙¤°℃℉€¥£¢¡®©_+]*$');
+      if (!charRegex.test(_.get(val)) ||
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts]: charRegex.test(val)

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts [hunk 2]
```diff
@@ -150,17 +150,13 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
...
+        !lwcsRegex.test(_.get(val)) ||
+        !upcsRegex.test(_.get(val)) ||
+        !numRegex.test(_.get(val)) ||
+        specRegex.test(_.get(val))) {
+        this.passwordError = _.get(this.resourceService.frmelmnts.lbl.passwd);
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts]: _.get(this.resourceService, 'femelmnts.lbl.passwd');

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts (hunk 1 above)
Comment (ArjunKhetia-NIIT) [reply src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts]: Done

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts (hunk 2 above)
Comment (ArjunKhetia-NIIT) [reply src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts]: Done

**Pull request merged**